CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -906,15 +906,16 @@ CF_EXPORT int _NS_open(const char *name, int oflag, int pmode) {
 
     DWORD dwCreationDisposition;
     switch (oflag & (O_CREAT | O_EXCL | O_TRUNC)) {
+      case O_CREAT | O_EXCL | O_TRUNC:
```

Comment:
I think that `O_TRUNC` is interesting, the behaviour between these two is different I believe.

Reply:
I don't think it is different, referencing https://pubs.opengroup.org/onlinepubs/9699919799/:
```
O_EXCL
If O_CREAT and O_EXCL are set, open() shall fail if the file exists. The check for the
existence of the file and the creation of the file if it does not exist shall be atomic
with respect to other threads executing open() naming the same filename in the
same directory with O_EXCL and O_CREAT set. If O_EXCL and O_CREAT are set, and
path names a symbolic link, open() shall fail and set errno to [EEXIST], regardless of
the contents of the symbolic link. If O_EXCL is set and O_CREAT is not set, the result
is undefined.
```
and 
```
O_TRUNC
If the file exists and is a regular file, and the file is successfully opened O_RDWR or
O_WRONLY, its length shall be truncated to 0, and the mode and owner shall be
unchanged. It shall have no effect on FIFO special files or terminal device files.
Its effect on other file types is implementation-defined. The result of using
O_TRUNC without either O_RDWR or O_WRONLY is undefined.
```
In the case of `O_CREAT | O_EXCL | O_TRUNC`, if a file exists, then O_EXCL causes open to fail regardless of `O_TRUNC`. If the file doesn't exist, then O_TRUNC doesn't do anything and is the same as `O_CREAT | O_EXCL`.